config/routes.rb

```diff
@@ -1,14 +1,11 @@
 Rails.application.routes.draw do
 
   resources :categories do
-    :memes do
+    resource :memes do
       :reviews
     end
   end
```

Comment:
Deeply nested resources here, lets make it shallow. We also need to use `only:` e.g `only: [:index, :destroy]`.
Otherwise, a use could navigate to a path and see a 500 error